funding/manager.go

```diff
@@ -4555,6 +4542,15 @@ func (f *Manager) getInitialForwardingPolicy(
 	copy(chanID, permChanID[:])
 
 	value, err := f.cfg.Wallet.Cfg.Database.GetInitialFwdingPolicy(chanID)
+
+	// The ErrChannelNotFound error is returned if there was no forwarding
+	// policy stored for this channel. In this case, we'll return the
+	// default forwarding policy. This should only happen when upgrading
+	// from an older version of lnd while a channel is still pending.
+	if errors.Is(err, channeldb.ErrChannelNotFound) {
```

Comment:
I guess this is a leftover from the previous PR, as we still can't tell that, we there's an `ErrChannelNotFound`, we don't it's due to the upgrading, or something wrong in db. If it's the latter, we will return a default policy, which may cause some hard-to-find bugs. In hindsight, we probably should've migrated the db by storing the default policy for these channels. Non-blocking tho as I don't we can do anything here atm.